mccodec/mc/configuration/clientbound_storecookie_config.mbt

```diff
@@ -0,0 +1,31 @@
+///|
+/// Store Cookie (configuration) packet (0x0A)
+/// Stores arbitrary data on client
+struct StoreCookieConfigS2C {
+  key : String // Cookie identifier, TODO: Identifier type
+  payload : FixedArray[Byte] // Cookie data (up to 5 kiB)
```

Comment:
inconsistent with other packets with binary payload. consider bytebuf or fixedarray

it's up to you but I personally prefer way 1 more